backend/Pipfile

```diff
@@ -13,6 +13,7 @@ bandit = "*"
 fastapi = "*"
 uvicorn = "*"
 email-validator = "*"
+pymongo = "*"
```

Comment:
Instead of pymongo, take a look at mongoengine: http://docs.mongoengine.org/

It is an ORM library built using pymongo to provide you with much better DX. (I personally love it and have been using it for a long time).